src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -699,6 +699,10 @@ function discoverController(
         return $scope.vis.getAggConfig().toDsl();
       });
     }
+
```

Comment:
Since the date histogram in the chart looks into this variable expected to be set by `visualize.js` we need to set it explicitly in the places we use `<visualization>` directly. This is currently only discover. I think in the long run with the new chart component in EUI we should rather use the chart component directly and not any part of the visualize stack anymore in Discover, since discover and visualize already have a ton of special handling just for this one chart.